generator/core_contracts.go

```diff
@@ -0,0 +1,101 @@
+package generator
+
+import (
+	"github.com/onflow/flixkit-go"
+)
+
+func getDefaultCoreContracts() flixkit.Contracts {
```

Comment:
I wasn't sure of a good way of handling the core contracts.  Even if they are loaded from a config file the values would be hardcode in the config file. Putting them in code skips that step. Since this is a lib and not a server.
The user can also pass in their own version of core contracts when they create the generator. Not sure the value add would be for a config file.